README.md

```diff
@@ -30,8 +30,10 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221179972-cd7f551a-85ba-4716-b18c-756af860639f.png)

**[Optional]**

- [ ] It would be nice if you fix your issues page link in the section shown in the image above to make your readme file descriptive and professional :+1: